### ISSUE TITLE
GitBook Editor(Web) attempts to open README.md even when `structure.readme` has been customized

### ISSUE THREAD

**Issue by Lin-Buo-Ren** [GitbookIO/feedback#209]
### Reproduce steps.
1. Customize `structure.readme` in `book.json`
1. Rename `README.md` to something else (e.g.  `README.markdown`)
1. Click the book's `Edit` button in the dashboard.

### What is the current behavior?
![loading master - gitbook editor - mozilla firefox_029](https://cloud.githubusercontent.com/assets/13408130/19364250/ff6fecb0-91bf-11e6-8b8f-39baf15de11f.png)

### What is the expected behavior?
* GitBook opens `structure.readme` instead of `README.md`
* GitBook should fallback to other Markdown file extensions if `README.md` is not available and `structure.readme` is undefined

#### For GitBook Editor
- Operating System?
    - Ubuntu 16.04 AMD64
- (Web) Browser version?
    - Mozilla Firefox 49.0 from Ubuntu software archive
- (Desktop) Editor version?
    - (Haven't attempted to reproduce issue) 6.2.1
- Was the bug present in previous versions?
    - Unknown

#### For GitBook.com
<https://www.gitbook.com/book/lin-buo-ren/ntoucs-physics-course-open-source-project>

**Comment by SamyPesse** [comment]
Yes, the web-editor currently do not support it, but the next version will (coming soon!).
